starlette/middleware/base.py

```diff
@@ -148,6 +136,8 @@ async def coro() -> None:
                     try:
                         await self.app(scope, receive_or_disconnect, send_no_error)
                     except Exception as exc:
+                        # import traceback
+                        # traceback.print_exc()
```

Comment:
If you uncomment those lines, you can see that we have the exceptions on the `app_exc`, but we don't reraise them at any point.